modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/Engine.scala

```diff
@@ -3,36 +3,374 @@
 
 package edu.gemini.seqexec.engine
 
-import monocle.Lens
-import monocle.macros.GenLens
+import edu.gemini.seqexec.engine.Event._
+import edu.gemini.seqexec.engine.Result.{PartialVal, PauseContext, RetVal}
+import edu.gemini.seqexec.model.Model.{CloudCover, Conditions, ImageQuality, Observer, Operator, Resource, SequenceState, SkyBackground, WaterVapor}
+import org.log4s.getLogger
 
-import edu.gemini.seqexec.model.Model.{Conditions, Operator}
+import scalaz.concurrent.Task
+import scalaz.{Applicative, Kleisli, MonadState, StateT}
+import scalaz._
+import Scalaz._
+import scalaz.stream.{Process, merge}
 
-/**
-  * A Map of `Sequence`s.
-  */
-final case class Engine(sequences: Map[Sequence.Id, Sequence])
+class Engine[D] {
```

Comment:
Where do `sequences` live now?

Reply:
Parameter `sequences` was not used. The one used has always lived in `Engine.State`